### PR TITLE
CMREP-87

### PR DESCRIPTION
**Pull request by juliacollins**
Note that I've hard-coded a change from a service type of "forking" to one of "simple" (see the changes to templates/systemd.erb). Any thoughts on whether I should make this configurable? The "forking" approach seems to be the source of problems with PID file permissions. Tests so far indicate that puma is being configured correctly on my dev VM as a "simple" service.

### CODE REVIEW

**Comment by mlaxer88** [comment]
Looks good to me!  I don't think it's necessary to have the systemd type be configurable for this module.  Type=simple seems to be the preferred method, according to puma 4 documentation: https://puma.io/puma4/file.systemd.html

File: README.md [hunk 1]
```diff
@@ -65,3 +66,25 @@ puma::nginxconfig {'myapp':
 ```
 
 This will create the necessary NGINX vhost and location configurations to serve a puma rails app.
+
+## Release Management
+
+The [`bump` gem](https://github.com/gregorym/bump) is used to manage version information in
+`CHANGELOG`, `metadata.json`, and Git tags. Currently this is a manual process. Insert a
+description of your changes at the top of the `CHANGELOG` file, leaving a blank line between
+your new description and the previous version entry. No header markup is necessary (`bump`
```
Comment (scott-lewis-nsidc) [inline README.md:75]: Just a consideration, do we want to keep it this way, or would it be better to keep the #UNRELEASED header as per our "current" workflow.  (I'm not sure I have a preference either way, other than the #UNRELEASED header makes it clear what's going on in terms of "this being a work in progress or non-bumped change entry).

File: manifests/params.pp [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # puma::params class
 class puma::params {
   $puma_user          = 'puma'
+  $www_user           = 'www-data'
```
Comment (scott-lewis-nsidc) [inline manifests/params.pp:4]: Does doing it this way prevent a client from specifying a different $www_user?  Or will this be overridden if they do?

File: README.md (hunk 1 above)
Comment (juliacollins) [reply README.md:75]: The instructions here take advantage of `bump`'s built-in capabilities. (I'm guessing this was a feature added in recent versions.) The "`## Unreleased`" pattern relies on extra `sed` processing after running `bump`. Going forward, I suggest that code bases using the "`## Unreleased`" pattern should actually switch to simply using the `bump` option to add the version number to the CHANGELOG. I agree that the "`## Unreleased`" text does provide feedback, but in my tests bump simply ignores it and inserts the version number heading where we want it.

File: manifests/params.pp (hunk 2 above)
Comment (juliacollins) [reply manifests/params.pp:4]: The default values are replaced by custom values if they're specified (we use user and group `vagrant`).

**Pull request merged**